dataland-backend/src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 springdoc.show-actuator=true
 springdoc.swagger-ui.oauth2RedirectUrl=https://${PROXY_PRIMARY_URL:-dataland-local.duckdns.org}/api/swagger-ui/oauth2-redirect.html
+springdoc.swagger-ui.oauth.use-pkce-with-authorization-code-grant=true
```

Comment:
This is also unused, I think.

Reply:
This one is needed to make the swagger UI use PKCE